Add Section about 'velocitas init' with package parameter (#121)
eclipse-velocitas/velocitas-docs: content/en/docs/concepts/lifecycle_management/velocitas_cli.md

...
```

#### Dynamically initialize packages NOT configured in `.velocitas.json`

> Packages which are not part of `.velocitas.json` can easily be initialized using the _package parameter_ `-p / --package` and the _specifier parameter_ `-s / --specifier`. The _specifier parameter_ can be either a git tag or a git hash. If the _specifier parameter_ is omitted the latest version of the specified package will be used automatically. After initialisation the package and it's resolved version will be written to `.velocitas.json`. If the package is already part of `.velocitas.json`, however the version is different the version will be automatically adapted to reflect the specified version.

Reviewer: ```suggestion
Packages can also easily be initialized or re-initialized using the _package parameter_ `-p / --package` and the _specifier parameter_ `-s / --specifier`. The _specifier parameter_ can be either a git tag or a git hash. If the _specifier parameter_ is omitted the latest version of the specified package will be used automatically. After initialisation the package and it's resolved version will be written to `.velocitas.json`. If the package is already part of `.velocitas.json`, however the version is different the version will be automatically adapted to reflect the specified version.
```
